src/main/kotlin/com/ampnet/reportserviceeth/service/Extensions.kt

```diff
@@ -13,8 +13,7 @@ import java.time.Instant
 import java.time.LocalDateTime
 import java.time.ZoneId
 import java.time.format.DateTimeFormatter
-import java.util.Locale
-import java.util.Optional
+import java.util.*
```

Comment:
Setup IntelliJ not to use wildcard imports. Preferences -> Editor -> Code Style -> Kotlin -> Use signle name import